addon/components/frost-action-link.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember'
+import layout from '../templates/components/frost-action-link'
+import _frostObjectDetails from './frost-object-details'
+import _ from 'lodash/lodash'
+
+export default Ember.Component.extend({
+  routing: Ember.inject.service('-routing'),
+
+  layout: layout,
+  classNames: ['frost-action-link'],
+  classNameBindings: ['isSelected'],
+
+  initContext: Ember.on('didInitAttrs', function () {
+    this.set('_frostObjectDetails', this.nearestOfType(_frostObjectDetails))
+  }),
+
+  isSelected: Ember.computed('actionLink', 'routing.currentRouteName', function () {
```

Comment:
Can we use https://github.com/emberjs/ember.js/blob/v2.4.0/packages/ember-routing-views/lib/components/link-to.js#L585 instead?